libbpf-tools/tcprtt.c

```diff
@@ -225,6 +263,12 @@ int main(int argc, char **argv)
 	if (err)
 		return err;
 
+	if ((env.laddr || env.raddr)
+		&& (!memcmp(env.laddr_v6, zero_addr_v6, sizeof env.laddr_v6) || !memcmp(env.raddr_v6, zero_addr_v6, sizeof env.raddr_v6))) {
+		fprintf(stderr, "It is not permitted to filter by both IPv6 and IPv4");
```

Comment:
Please add '\n' as the end of error string. Maybe other places need '\n' as well. Please double check.